components/docs-chef-io/content/automate/external_postgres_upgrade_steps.md

```diff
@@ -0,0 +1,253 @@
++++
+title = "External PostgreSQL Upgrade"
```

Comment:
Parts of this document reference PostgreSQL 13.3 and parts reference 13.4. Is that intentional?

Reply:
We're using 13.4 in this doc.